mock-recipe-server/bin/upload.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -eu
+
+BUILD_DIR=$1
+S3_BUCKET=$2
+
+aws s3 rm --recursive "s3://$S3_BUCKET/"
+aws s3 cp --recursive "$BUILD_DIR" "s3://$S3_BUCKET/" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers
```

Comment:
It might make more sense to use `aws s3 sync --delete "$BUILD_DIR" "s3://$S3_BUCKET/" --grants read=uri=http://acs.amazonaws.com/groups/global/AllUsers`, that way you don't re-copy existing files and only delete those which have been deleted.